pkg/routes/logs_test.go

```diff
@@ -37,11 +40,19 @@ func TestPreCheckLogFileNameLength(t *testing.T) {
 	}
 
 	// check file with normal name which does exist.
-	_, err := os.Create(normalFileName)
+	dir, err := ioutil.TempDir("", "logs")
```

Comment:
Maybe a note why a directory is created here? I guess this isn't need on linux but required for windows?

Reply:
We can revert this part. I've added this so the file would be created in the temp directory, rather than the current directory, so there wouldn't be any leaked files in the current directory (this happened because the created file could not be removed because it was still open, which is now solved by the ``defer f.Close()`` below).